pkg/api/api.go

```diff
@@ -0,0 +1,25 @@
+package api
+
+import "kvstore/pkg/errors"
+
+var (
+	Data = make(map[string]string)
```

Comment:
I agree. It was public because of my false understanding of the scope exposure in test files.
I thought about the struct but can't find a strong reason yet to have it as non-singelton. what od you think?